packages/rum-recorder/src/domain/rrweb/observer.ts

```diff
@@ -431,83 +429,7 @@ function initFontObserver(cb: FontCallback): ListenerHandler {
   }
 }
 
-function mergeHooks(o: ObserverParam, hooks: HooksParam) {
```

Comment:
Could we also remove the hook setter / resetter logic?

Reply:
`hookSetter` in `utils` isn't related to the `hooks` option, and it is still used to observe inputs